IntegrationTests/IntegrationScoped.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace IntegrationTests
+{
+    public class IntegrationScoped<TStrategy> where TStrategy : IScopedStrategy, new()
+    {
+
+        protected readonly TStrategy Strategy = new TStrategy();
+
+        [TestInitialize]
+        public void Setup()
+        {
+            Strategy.Setup().Wait();
```

Comment:
yeah it does : Xunit uses IUseFixture<T> which does exactly what you are trying to do